runtime/jcl/common/unsafe_mem.c

```diff
@@ -146,6 +145,7 @@ unsafeFreeMemory(J9VMThread* vmThread, void* oldAddress)
 		J9_LINKED_LIST_REMOVE(vmThread->javaVM->unsafeMemoryListHead, memBlock);
 		omrthread_monitor_exit(mutex);
 #if defined(J9VM_OPT_SNAPSHOTS)
+		J9JavaVM *vm = vmThread->javaVM;
```

Comment:
this will break windows, if a variable is not defined at the beginning of a scope. Don't worry about having multiple `#if defined(..)` blocks. Just define the variable at the start of the scope.

Reply:
Fixed in https://github.com/eclipse/openj9/compare/74d9e544302db2e57a039f6bc3250808729d938a..a001f260c87796da222a7d5f0aa36ca27e7d5b69